staging/src/k8s.io/legacy-cloud-providers/hack/tools.go

```diff
@@ -0,0 +1,23 @@
+//go:build tools
```

Comment:
for this to vendor, I think this has to be a package we actually depend on

Reply:
@liggitt should i just go back to what i had before? :) not seeing the point of a extra hack directory and tools.go and then adding a dependency to this package when the tools.go just has what was in the original file...

Reply:
going back to what i had before @liggitt